app/views/plates/_pooled_wells_samples_tab.html.erb

```diff
@@ -0,0 +1,11 @@
+<dl id="samples-information" class="card-body">
+  <% plate_presenter.wells.each do |well| -%>
+    <% next if well.aliquots.empty? %>
+    <dt class="col-sm-1"><%= well.location %> <%= state_badge(well.state) %></dt>
+    <dd class="col-sm-11"><ul class="list-inline">
+    <% well.upstream_wells.each do |parent_well| -%>
+      <li class="list-inline-item"><%= parent_well.aliquots.first.sample.sanger_sample_id %> (source:
```

Comment:
each here instead of first

Reply:
parent wells have single aliquots for LRC PBMC Pools plate. It is OK for this story.